index.html

```diff
@@ -1,19 +1,143 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>Ian Belknap - Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
+  <link href="https://fonts.googleapis.com/css?family=Aleo" rel="stylesheet">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+
+  <div id="top" class="container">
+
+    <header>
+
+      <h1>Ian Belknap</h1>
+      <contact>
+        <div class="bigredbrackets">{</div>
+        <p>
+          e - mail<br>
+          phone<br>
+          fax<br>
+        </p>
+        <p>
+          :<br>
+          :<br>
+          :<br>
+        </p>
+        <p>
+          <a href="mailto:ian84be@gmail.com">ian84be@gmail.com</a><br>
+          716 - 430 - 9838<br>
+          404 - lol - jkjk<br>
+        </p>
+        <div class="bigredbrackets">}</div>
+      </contact>
+
+      <nav>
+        <a href="#intro">Introduction</a>
+        <a href="#skills">Skill Set</a>
+        <a href="#exp">Professional Experience</a>
+        <a href="https://ian84be.github.io/portfolio-website/" target="_blank">Portfolio</a>
+      </nav>
+
+    </header>
+
+    <!-- INTRO SECTION -->
+    <article id="intro">
+      <h2>Hello World</h2>
+      <p>
+        I am a resume for Ian Belknap, the hardest workingest employee you ever did would or will employ, thank you
+        very much!
+      </p>
+      <p>
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quas ad natus obcaecati minus dolorum rem voluptates
+        repellendus facere mollitia doloribus omnis repudiandae aliquid harum facilis, quidem exercitationem cumque
+        totam a?
+      </p>
+      <p>
+          To make the matter of pleasures to be cast to the effeminacy of these sorrows, the pains of birth, all been blinded by the less easy to reject a thing of this world, and when, indeed, the practice of the island from?
+      </p>
+      <a href="#top">^^^ top ^^^</a>
```

Comment:
nice feature for allowing users to go back to the top of the page at any section: the improved version of this would be to make a fixed nav that follows the user down instead of including these links in the flow of your page. still, very nice implementation